package.json

```diff
@@ -28,6 +28,8 @@
     "bundle": "webpack --config webpack.config.js --mode=production",
     "copy": "cp package.json dist/ && cp README.md dist/ && cp -r license_output dist/",
     "production": "npm run bundle && npm run copy",
+    "version:patch": "npm version patch --no-git-tag-version",
```

Comment:
Does this command increase path version number only, what about major/minor versions? Just curious about the plan

Reply:
we can add similar scripts `"version:minor": npm version minor`, `"version:major": npm version major`
I added this one as this is the most common one.